cscs-checks/tools/profiling_and_debugging/notool.py

```diff
@@ -75,16 +68,52 @@ def __init__(self):
             f'echo GNU_VERSION=$GNU_VERSION',
             f'echo PGI_VERSION=$PGI_VERSION',
             f'echo INTEL_VERSION=$INTEL_VERSION',
+            f'echo INTEL_COMPILER_TYPE=$INTEL_COMPILER_TYPE',
             f'echo CRAY_AOCC_VERSION=$CRAY_AOCC_VERSION',
         ]
         self.reference = {'*': {'elapsed_time': (0, None, None, 's')}}
-        if self.lang == 'C++':
-            self.reference_lang = (0.38, -0.6, None, 's')
-        elif self.lang == 'F90':
-            self.reference_lang = (0.17, -0.6, None, 's')
+
+    @run_before('compile')
+    def prgEnv_nvidia_workaround(self):
+        # {{{ CRAY_MPICH_VERSION
+        # cdt/20.08 cray-mpich/7.7.15 cray, crayclang, gnu, intel, pgi
+        # cdt/21.02 cray-mpich/7.7.16 cray, crayclang, gnu, intel, pgi
+        # cdt/21.05 cray-mpich/7.7.17 crayclang, gnu, intel, pgi, *nvidia*
+        #
+        # cpe/21.04 cray-mpich/8.1.4 AOCC, CRAY, CRAYCLANG, GNU, INTEL, NVIDIA
+        # cpe/21.05 cray-mpich/8.1.5 AOCC, CRAY, CRAYCLANG, GNU, INTEL, NVIDIA
+        # cpe/21.06 cray-mpich/8.1.6 AOCC, CRAY, CRAYCLANG, GNU, INTEL, NVIDIA
+        # }}}
+        envname = self.current_environ.name
+        self.cppflags = None
```

Comment:
I think this could solve it.
```suggestion
        self.cppflags = ''
```